index.php

```diff
@@ -108,6 +108,10 @@
 	 * @deprecated Use O_O instead
 	 */
 	$hyphaLanguage = $O_O->getInterfaceLanguage();
+	$dict = Language::getDictionaryByLanguage($hyphaLanguage);
+	if (array_key_exists('_locale', $dict)) {
```

Comment:
Why not use `__()` here? The only caveat I can think of is that it would return `_locale` if not found, but that seems easy to check?

Reply:
fixed